portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsList.testcase

```diff
@@ -83,4 +83,58 @@ definition {
 		ACSegments.deleteSegment(searchTerm = "Static Segment Test 2");
 	}
 
+	@description = "Search for specific segment"
+	@priority = "5"
+	test CanShowSegmentList {
+		JSONUser.addUser(
+			userEmailAddress = "userea@liferay.com",
+			userFirstName = "userfn",
+			userLastName = "userln",
+			userScreenName = "usersn");
+
+		ACDXPSettings.connectDXPtoAnalyticsCloud();
+
+		Navigator.openURL();
+
+		Pause(locator1 = "5000");
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "Andre DXP 7.4");
```

Comment:
The property name should be Liferay DXP?

Reply:
thank you @joshitaki